socrates/lib/site/views/index.jade

```diff
@@ -60,61 +60,66 @@ block content
               h5
                 +extlink('http://www.codefreeze.fi/', 'CodeFreeze')
                 br
-                small 3 - 4 Feb 2015, Kiilopää, Finland
+                small 11 - 12 Jan 2016 , Kiilopää, Finland
             .col-sm-6
               h5
-                +extlink('http://socrates-conference.es/', 'SoCraTes Canaries 2015')
+                +extlink('http://www.socrates-conference.es/', 'SoCraTes Canaries')
                 br
-                small 26 Feb - 1 Mar 2015, Tenerife
+                small 10 - 13 Mar 2016, Gran Canaria
             .col-sm-6
               h5
-                +extlink('http://2015.itakeunconf.com/', 'I T.A.K.E. Unconference')
+                +extlink('http://itakeunconf.com/', 'I T.A.K.E. Unconference')
                 br
-                small 28 - 29 May 2015, Bucharest
+                small 19 - 20 May 2016, Bucharest
             .col-sm-6
               h5
-                +extlink('http://socratesuk.org/', 'SoCraTes UK 2015')
+                +extlink('http://socratesuk.org/', 'SoCraTes UK')
                 br
-                small 4 - 7 June 2015, Farncombe Estate
+                small 2 - 5 June 2016, Farncombe Estate
             .col-sm-6
               h5
                 +extlink('http://socrates-day.ch/', 'SoCraTes Day Switzerland')
```

Comment:
Is that really better? Particularly as not all the conferences include date in names and some of them have not decided on dates yet.